content/topics/java-specific/spring-boot-rest/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Spring Boot REST tutorial
+ready: true
+---
+
+This document highlights the way in which the various types of endpoints are created in spring api's
+
+## Here is a list of resources where you can find tutorials on the endpoint types
+
+# GetMapping
+- [GET Endpoint](http://zetcode.com/spring/getmapping/).
+
+# PostMapping
+- [POST Endpoint](https://howtodoinjava.com/spring5/webmvc/controller-getmapping-postmapping/).
+
+# PutMapping
+- [PUT Endpoint](https://www.sourcecodeexamples.net/2019/10/putmapping-spring-boot-example.html).
+
+# DeleteMapping
+- [DELETE Endpoint](http://zetcode.com/spring/deletemapping/).
+
+## How can a developer test endpoints
+
+- [PostMan](https://www.postman.com/).
+
```

Comment:
Always add a resource section linking all the url you used to complete the topic